tests/MaintenanceModeTest.php

```diff
@@ -31,3 +30,8 @@
     $this->withoutExceptionHandling()
         ->get('http://acme.localhost/foo');
 });
+
+class MaintenanceTenant extends Tenant
+{
+    use MaintenanceMode;
+}
```

Comment:
```suggestion
}

```
empty line at the end